build/Targets/Settings.targets

```diff
@@ -148,6 +149,16 @@
           </PropertyGroup>
         </When>
 
+        <When Condition="'$(IsDev15RC)' == 'true'">
+          <PropertyGroup>
+            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25825-RC</VisualStudioBuildToolsNuGetPackagePath>
+            <!-- Currently, the Dev15 Preview 5 SDK won't install binaries onto an RC build, an RC build won't (normally) build without a <Prerequisites> 
+                 element, but the Willow installer technology doesn't know how to install VSIXes that *do* have a a <Prerequisites> element :(
+                 In order to be able to both build and install on RC, we'll disable schema validation and *not* add any <Prerequisites> elements. -->
+            <BypassVsixValidation>true</BypassVsixValidation>
+          </PropertyGroup>
```

Comment:
@OmarTawfik just a heads up this will cause a merge conflict or build conflict when it makes its way to master due to some other changes i'm making.  Drop by when this happens and I can show you the quick fix.